### DISCUSSION TITLE
Soft Unbrick/Uninstall not working

### DISCUSSION THREAD

**Discussion by rpkilgore** [roleoroleo/yi-hack-Allwinner-v2#389] [General]
I have no idea what has changed or if Kami updated their software or if it's something I'm doing incorrectly, but the unbrick process doesn't appear to be working.
I tried it on a r35gb and it did not factory reset anything.  The camera just comes up as normal.  Almost as if it's not reading the backup_r35gb file.  This might also explain why the unbrick isn't working on my other y28gb cam..

Anyone know of anything that has changed recently to make the unbrick process not work?  Pretty sure I followed the directions correctly.  Copy the mtdblock4.bin from the backup I made to the unbrick/r35gb folder, run the build.sh, then extract the backup_r35gb to the root of the sd card and boot up the camera, then wait for the blue lights.. blue lights never come, and the cam just boots up as normal as if i just put in a clean sd card.

**Comment by uklatt** [comment]
I had similar problems with y21ga.
I needed to use a different SD card.
Uwe

**Comment by rpkilgore** [comment]
I've tried 3 different sd cards, 2 32g, and one 8g.  I've reformatted them via windows, and the kami app, but still no luck on getting it to flash back to factory.

**Comment by rpkilgore** [comment]
Wanted to update to anyone that might be having the same issue.  After digging through the other issues and reading through lots of discussions I came across [this post ](https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/305#issuecomment-987908753) It's a windows executable that will build all the possible flash files.  After trying the backup_y28ga, home_y28ga, and kernel_y28ga I finally just decided to go all in with the one_y28ga and that brought my cam back to life!  I'm sure this should probably be a last resort, but it worked for me..   Hope this helps someone else.

**Comment by roleoroleo** [comment]
Remember that you cannot apply the same process twice with the same file (consecutively).
If you restore the cam with backup_y28ga, then you can't restore it again with the same file.
Because the bootloader saves the hash of the last file used and, if the hash doesn't change, it doesn't load the file.
You need another backup file with at least 1 different byte.

**Reply by rpkilgore** [reply]
Ahh well that makes sense.  That leads me to my next question then,  how can I mount the bin files and make changes to them?  Do you use chroot or something? or can they be mounted and modified slightly?

**Reply by roleoroleo** [reply]
Check the scripts in "unbrick" folder.
